CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for review when someone
+# opens a pull request.
+*       @blootsvoets @nivemaham @fnobilia
```

Comment:
We need to find a tradeoff.

For `hotfix` we can use a separate branch as described [here](https://www.atlassian.com/git/tutorials/comparing-workflows#gitflow-workflow), but not sure how dangerous it could be.

Reviewing the documentation will slow the process, you are right. But my way of documenting code could not be clear to you, and viceversa. Having a documentation that is not clear is useless (I have a little bit accentuated here). In the end, also this thing could be worthy.